apis/cloud.redhat.com/v1alpha1/jobinvocation_types.go

```diff
@@ -0,0 +1,67 @@
+/*
+
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// JobInvocationSpec defines the desired state of JobInvocation
+type JobInvocationSpec struct {
+	// Name of the ClowdApp who owns the jobs
+	AppName string `json:"appName"`
+
+	// Jobs is the set of jobs to be run by the invocation
+	Jobs []string `json:"jobs"`
+}
+
+type JobResult struct {
+	Name      string `json:"name"`
+	Completed bool   `json:"completed"`
+	Attempts  int    `json:"attempts"`
+	StdOut    string `json:"stdout"`
```

Comment:
Is there a `StdErr` in this too? If so, perhaps it would be better to call it "output" - I didn't look further down yet - but we probably want to be careful how much data we put into this. I realise we will want to give people an output so they can debug - but we may need to think about this a little more.

Reply:
This was some noodling that I dropped in so we could discuss what we needed in here. I agree that it's currently an overload